Story/troubleshooting section pt2 (#118)
vmware/vmware-aria-operations-integration-sdk: README.md

<details>

>   <summary><h3>How can change the container registry mp-build uses?</h3></summary>

Reviewer: ```suggestion
  <summary><h3>How can I change the container registry for my project?</h3></summary>
```

</details>

<details>

>   <summary><h3> How do I change the log level (SDK tools)?</h3></summary>

Reviewer: ```suggestion
  <summary><h3> How do I change the log level of mp-init, mp-test, or mp-build?</h3></summary>
```

  All SDK tools read the LOG_LEVEL environment variable to set the log level of their console output. For example, to set log level
  to debug to see a verbose output of the any of the CLI tools we can set the LOG_LEVEL variable:

  For Linux and macOS
  ```shell
  LOG_LEVEL=debug mp-build
  ```
  For Windows
  ```
  set LOG_LEVEL=debug
  mp-build
  ```
  For Windows, set the log level back to `info` after debugging.

  All logs can be seen in the command line, but they are also saved in  `logs/build.log` with `debug` log level. CLI tools generate logs

>   for `debug`, `warn`, `info`, and `error` level.

Reviewer: This is confusing, because I _think_ it's just referring to the specific example of `mp-build` at `debug` level, but it reads more generally. I think we can probably remove most of it and just say what the log level options are.

  Using the collection error message along with the `adapter.log` can help trace the cause of the issue.
</details>

<details>

>   <summary><h3>Unable to build pak file after running mp-build?</h3></summary>

Reviewer: ```suggestion
  <summary><h3>mp-build returns 'Unable to build pak file'</h3></summary>
```

  In most cases, this error indicates issues with building the container image. The most probable causes are:

  1. Syntax error in the Dockerfile:

  ```
  mp-build
  Building adapter [Finished]
  Unable to build pak file
  ERROR: Unable to build Docker file at /Users/user/code/management-packs/test:

>    {'message': 'dockerfile parse error line 6: unknown instruction: SYNTAX'}

Reviewer: This message seems to indicate case 2 (unknown instruction). Are these distinct enough to have separate cases? They have the same solution below.

  ```

  2. Unknown Instruction :

  ```
  mp-build
  Building adapter [Finished]
  Unable to build pak file
  ERROR: Unable to build Docker file at /Users/user/code/aria_ops/management-packs/test:
   {'message': 'dockerfile parse error line 7: unknown instruction: COP'}

  ```
  3. A command  inside the Dockerfile failed:

  ```
  mp-build
  Building adapter [Finished]
  Unable to build pak file
  ERROR: Unable to build Docker file at /Users/user/code/management-packs/test:
   The command '/bin/sh -c pip3 install -r adapter_requirements.txt --upgrade' returned a non-zero code: 1
  ```

>   The solution for cases 1 and 2 is to fix the typo/command by editing the Dockerfile. For case 3, however, the solution might need to be evident at first sight. Since the error

Reviewer 2: `might need to be evident` -> `might not be evident`

>   comes from building the image itself, we can run `docker build .` directly on the project's root directory and look at the stack trace for clues.

Reviewer: ```suggestion
  comes from building the image itself, we can run `docker build .` in the project's root directory and look at the stack trace for clues.
```
